app/views/patients/_new_patient.html.erb

```diff
@@ -24,8 +24,8 @@
       </div>
 
       <div class="col-sm-2">
-        <%= f.form_group :spanish, label: { text: 'Spanish only' } do %>
-          <%= f.check_box :spanish, label: '' %>
+        <%= f.select :language, options_for_select(language, patient.language ),
+          label: 'Preferred Language', autocomplete: 'off' %>
         <% end %>
```

Comment:
I think this extra `end` is what's making the tests choke. (This is just a select instead of a `form_group ... do` block, so no need to close it anymore.) No promises but I bet this will get CI to either pass or mostly pass.